template/koa-rest-api/src/listen.ts

```diff
@@ -1,3 +1,5 @@
+import './register';
+
```

Comment:
Making this explicit now because you can run into issues if `./app` is not imported first.